clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -324,62 +327,6 @@ private long listOffset(TopicPartition partition, long timestamp) {
     }
 
     /**
-     * If any partition from previous fetchResponse contains OffsetOutOfRange error and
-     * the defaultResetPolicy is NONE, throw OffsetOutOfRangeException
-     *
-     * @throws OffsetOutOfRangeException If there is OffsetOutOfRange error in fetchResponse
-     */
-    private void throwIfOffsetOutOfRange() throws OffsetOutOfRangeException {
```

Comment:
Do I understand correctly that we used to throw an exception with all the topic partitions with the issue whereas now we will throw the exception for a single topic partition? And, if so, do we care and are we missing test coverage that would catch this change in behaviour?